.eslintrc.cjs

```diff
@@ -25,16 +25,13 @@ module.exports = {
     'plugin:storybook/csf',
   ],
   ignorePatterns: ['./leather-styles'],
-  plugins: ['react', 'react-hooks', '@typescript-eslint', 'deprecation'],
```

Comment:
The deprecation plugin is itself deprecated as its included in ts-eslint rules. Also throwing incorrect chrome deprecation error.